### PR TITLE
Improve browser tests documentation, with more Sauce Labs instructions

### PR DESCRIPTION
**Pull request by cfarm**
see https://github.com/cfpb/development/issues/131 for background

## Additions

-

## Removals

-

## Changes

-

## Testing

1.

## Notes

-

## Todos

-

## Checklist

- [ ] PR has an informative and human-readable title
- [ ] Changes are limited to a single goal (no scope creep)
- [ ] Code can be automatically merged (no conflicts)
- [ ] Code follows the standards laid out in the [CFPB development guidelines](https://github.com/cfpb/development)
- [ ] Passes all existing automated tests
- [ ] Any _change_ in functionality is tested
- [ ] New functions are documented (with a description, list of inputs, and expected output)
- [ ] Placeholder code is flagged / future todos are captured in comments
- [ ] Visually tested in supported browsers and devices (see checklist below :point_down:)
- [ ] Project documentation has been updated
- [ ] Reviewers requested with the [Reviewers tool](https://help.github.com/articles/requesting-a-pull-request-review/) :arrow_right:

### Browsers

- [ ] Chrome on desktop
- [ ] Firefox
- [ ] Safari on macOS
- [ ] Edge
- [ ] Internet Explorer 9, 10, and 11
- [ ] Safari on iOS
- [ ] Chrome on Android

### Accessibility

- [ ] Keyboard friendly
- [ ] Screen reader friendly

### Other

- [ ] Is useable without CSS
- [ ] Is useable without JS
- [ ] Flexible from small to large screens
- [ ] No linting errors or warnings
- [ ] JavaScript tests are passing

### CODE REVIEW

File: .env_SAMPLE [hunk 1]
```diff
@@ -155,7 +155,7 @@ export TEST_HTTP_HOST=localhost
 #export SAUCE_USERNAME=<sauce_username>
 #export SAUCE_ACCESS_KEY=<sauce_access_key>
 #export SAUCE_SELENIUM_URL=localhost:4445/wd/hub
-#export SAUCE_TUNNEL=<sauce_tunnel_id>
+#export SAUCE_TUNNEL="Browser tests for cf.gov"
```
Comment (anselmbradford) [inline .env_SAMPLE]: Why the change from `<…>` to a string?

File: docs/browser-acceptance-tests.md [hunk 2]
```diff
@@ -118,23 +118,36 @@ Scenario: User logs in
...
-3. Reload the settings with `source .env`.
-
-4. Run the tests with `gulp test:acceptance --sauce`.
-
-5. Monitor progress of the tests
+### For new Sauce Labs users
+
+1. [Follow the instructions here](https://github.com/cfpb/development/blob/master/guides/browser-testing-with-sauce-labs.md) in the "Sign up for saucelabs.com" and "Set up Sauce Connect to test URLs on local and dev servers" sections. The remaining sections are not required to run cfgov-refresh browser tests, but are recommended if you want to use Sauce Labs to manually QA in supported browsers.
+1. Edit your cfgov-refresh `.env` file. Uncomment the `export` statements in the "Sauce Labs section of the file, and update them as follows:
+    -  `SAUCE_USERNAME`: Your saucelabs.com username
```
Comment (anselmbradford) [inline docs/browser-acceptance-tests.md]: TRIVIAL: Extra space before dash

**Comment by anselmbradford** [comment]
::bump:: to this one. Looks like it would be pretty easy to wrap up.

File: docs/browser-acceptance-tests.md (hunk 2 above)
Comment (cwdavies) [reply docs/browser-acceptance-tests.md]: I committed a change to remove extra space before dash for SAUCE_USERNAME in browser-acceptance-tests.md

**Comment by cwdavies** [comment]
@cfarm Can this 2019 pull request be merged or closed? It is still a work in progress.
